packages/workers-shared/router-worker/src/worker.ts

```diff
@@ -3,19 +3,46 @@ import { PerformanceTimer } from "../../utils/performance";
 import { setupSentry } from "../../utils/sentry";
 import { mockJaegerBinding } from "../../utils/tracing";
 import { Analytics, DISPATCH_TYPE, STATIC_ROUTING_DECISION } from "./analytics";
-import { applyConfigurationDefaults } from "./configuration";
+import {
+	applyEyeballConfigDefaults,
+	applyRouterConfigDefaults,
+} from "./configuration";
 import type AssetWorker from "../../asset-worker";
 import type {
+	EyeballRouterConfig,
 	JaegerTracing,
 	RouterConfig,
 	UnsafePerformanceTimer,
 } from "../../utils/types";
 import type { ColoMetadata, Environment, ReadyAnalytics } from "./types";
 
+const limitedResponse = `<html>
```

Comment:
styling and wording approved now? It's a bit of a departure from our usual styling.